package.json

```diff
@@ -18,6 +18,7 @@
     "@bulatdashiev/svelte-slider": "^1.0.3",
     "@iconify-json/mdi": "^1.1.63",
     "@iconify-json/ri": "^1.1.17",
+    "@melt-ui/svelte": "^0.70.0",
```

Comment:
Preprocessor was not used. When I tried it I got an error message from melt ui on the console suggesting it was not being used even though it was installed and configured in `svelt.config.js` (See commit: https://github.com/microbit-foundation/ml-trainer/pull/171/commits/3f3e21b4ca875e9b89fa372a831d68d9ba0e5a49)